feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/presentation/account/details/mixin/PolkadotVaultWalletDetailsMixin.kt

```diff
@@ -0,0 +1,59 @@
+package io.novafoundation.nova.feature_account_impl.presentation.account.details.mixin
+
+import io.novafoundation.nova.common.list.GroupedList
+import io.novafoundation.nova.common.list.headers.TextHeader
+import io.novafoundation.nova.common.resources.ResourceManager
+import io.novafoundation.nova.common.utils.flowOf
+import io.novafoundation.nova.feature_account_api.domain.model.MetaAccount
+import io.novafoundation.nova.feature_account_api.domain.model.asPolkadotVaultVariantOrThrow
+import io.novafoundation.nova.feature_account_api.presenatation.account.details.ChainAccountActionsSheet.AccountAction
+import io.novafoundation.nova.feature_account_api.presenatation.account.polkadotVault.config.PolkadotVaultVariantConfigProvider
+import io.novafoundation.nova.feature_account_impl.domain.account.details.AccountInChain
+import io.novafoundation.nova.feature_account_impl.domain.account.details.WalletDetailsInteractor
+import io.novafoundation.nova.feature_account_impl.presentation.account.details.mixin.common.AccountFormatterFactory
+import io.novafoundation.nova.feature_account_impl.presentation.account.details.mixin.common.hasAccountComparator
+import io.novafoundation.nova.feature_account_impl.presentation.account.details.mixin.common.polkadotVaultAccountTypeAlert
+import io.novafoundation.nova.feature_account_impl.presentation.account.details.mixin.common.polkadotVaultTitle
+import io.novafoundation.nova.feature_account_impl.presentation.account.details.mixin.common.withChainComparator
+import io.novafoundation.nova.feature_account_impl.presentation.account.details.model.AccountTypeAlert
+import io.novafoundation.nova.feature_account_impl.presentation.common.chainAccounts.AccountInChainUi
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.first
+
+class PolkadotVaultWalletDetailsMixin(
```

Comment:
`PolkadotVaultWalletDetailsMixin` and `ParitySignerWalletDetailsMixin` are almost identical and can be combined